### PR TITLE
Reply 200 to conntest

### PR DESCRIPTION
**Pull request by Maritoguionyo**

### CODE REVIEW

**Comment by mkwcat** [comment]
This is nice actually because my Wii was failing the connection test yesterday when I was testing DNS

File: nas/main.go [hunk 1]
```diff
@@ -82,6 +82,13 @@ func handleRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Reply 200 to conntest
+	if strings.HasPrefix(r.Host, "conntest.") {
+		// Respond with a 200 status code
+		replyHTTPError(w, 200, "ok")
```
Comment (MikeIsAStar) [inline nas/main.go:88]: It seems unusual to pass a status code of 200 to the `replyHTTPError` function.

Comment (Maritoguionyo) [reply nas/main.go:88]: Could renaming replyHTTPError to replyHTTPCode be fine?

Comment (mkwcat) [reply nas/main.go:88]: I've cleaned this up already in the latest commit, it returns the test html page now

**Pull request merged**